riscv/plic.cc

```diff
@@ -259,7 +262,7 @@ bool plic_t::context_write(plic_context_t *c,
     }
     default:
       ret = false;
-      update = true;
+      update = false;
```

Comment:
Delete this line.  `update` is initialized to false, so setting it to false here is redundant.

Reply:
I think it's more readable by assign true explicitly rather than deduce its value by source code context, so I add this line.  

I will make some change.👌🏻